css/stylesheet.css

```diff
@@ -193,9 +202,37 @@ blockquote {
   margin-top: 0;
 }
 
+ol {
+  margin: 30px auto;
+  border: 3px solid white;
+  width: 55%;
+  padding: 30px 50px;
+  font-size: 20px;
```

Comment:
Now that we've gone through the responsive week - I would consider switching these to use `em`s or `rem`s.